src/client/_clientwindow.py

```diff
@@ -870,12 +874,18 @@ def saveWindow(self):
         util.settings.setValue("geometry", self.saveGeometry())
         util.settings.endGroup()
 
+    def on_actionSetAutoJoinChannels(self):
```

Comment:
It'd be nice to give this a more descriptive name. How about 'show_autojoin_settings_dialog'?

Reply:
Ok, done